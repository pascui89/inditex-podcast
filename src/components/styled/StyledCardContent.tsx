import {
    CardContent,
    styled,
  } from '@mui/material';

export const StyledCardContent = styled(CardContent)`
    max-width: 100%;
    border-top: 0.1em solid rgba(0, 0, 0, 0.14);
    padding: 0.5em;
    margin-top: 4px;
`;

export const StyledCardContentPaddingNone = styled(CardContent)`
    padding: 0;
    padding-bottom: 0 !important;
`;