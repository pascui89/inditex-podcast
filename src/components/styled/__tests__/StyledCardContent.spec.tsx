import React from 'react';
import { render } from '@testing-library/react';
import { StyledCardContent, StyledCardContentPaddingNone } from '../StyledCardContent';
import '@testing-library/jest-dom/extend-expect';

global.React = React;

describe('StyledCardContent', () => {

  it('should render StyledCardContent', () => {
    const { container } = render(<StyledCardContent/>);
    expect(container).toMatchSnapshot();
  });

  it('should render StyledCardContentPaddingNone', () => {
    const { container } = render(<StyledCardContentPaddingNone/>);
    expect(container).toMatchSnapshot();
  });
});
