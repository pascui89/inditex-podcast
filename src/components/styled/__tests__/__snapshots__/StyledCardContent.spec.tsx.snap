// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledCardContent should render StyledCardContent 1`] = `
<div>
  <div
    class="MuiCardContent-root css-tjzwvn-MuiCardContent-root"
  />
</div>
`;

exports[`StyledCardContent should render StyledCardContentPaddingNone 1`] = `
<div>
  <div
    class="MuiCardContent-root css-1aqrhzj-MuiCardContent-root"
  />
</div>
`;
